src/DGtal/geometry/surfaces/ArithmeticalDSSComputerOnSurfels.h

```diff
@@ -60,8 +62,10 @@ namespace DGtal
   // class ArithmeticalDSSComputerOnSurfels
   /**
    * \brief Aim: This class is a wrapper around ArithmeticalDSS that is devoted
-   * to the dynamic recognition of digital straight segments (DSS) along any
-   * sequence of 3D surfels.
+   * to the dynamic recognition of digital straight segments (DSS) along a 
+   * sequence of surfels lying on a slice of the digital surface (i.e., the 
+   * orthogonal direction of all surfels belong to a same plane, most pairs
```

Comment:
```suggestion
   * orthogonal direction of all surfels are aligned to the same axis, most pairs
``` 
?

Reply:
they are actually not aligned to the same axis, but they are in the same 2D plane (the slice)

Reply:
sure the suggests was to mention the slice defined from axis but we can let it